course_catalog/constants.py

```diff
@@ -26,13 +26,13 @@ class ResourceType(Enum):
 class AvailabilityType(Enum):
     """
     Enum for Course availability options dictated by edX API values.
-    All OCW courses should be set to "Current"
+    All OCW courses should be set to "Available Now"
     """
 
-    current = "Current"
+    available_now = "Available Now"
     upcoming = "Upcoming"
     starting_soon = "Starting Soon"
-    archived = "Archived"
+    prior = "Prior"
```

Comment:
Some courses from mitx will have availabilities of "Current" and "Archived" when imported.  If the point of this PR is to display them differently, there is some front-end JS code to do that [here](https://github.com/mitodl/open-discussions/blob/e710b62e1f39d02ec1dc065d745a87be6551b89b/static/js/lib/courses.js#L11-L20).  Or you could transform the values during the [course data import](https://github.com/mitodl/open-discussions/blob/e710b62e1f39d02ec1dc065d745a87be6551b89b/course_catalog/task_helpers.py#L320).

Reply:
Ah, thanks for the data context. Given that, to minimize change, I think we just keep the Availability constants on the backend as is and simply make a note that the values are coming in that way, what the mapping is, and the conversion happening in the frontend.